### PR TITLE
quickhash 3.3.4

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-cask-pr`.

### CODE REVIEW

**Review by bevanjkay — DISMISSED** [review]

**Comment by bevanjkay** [comment]
@samford The latest update to this app requires that we find the date that the application was last updated, for matching the internal `.dmg` container inside of the downloaded `zip` file.
This regex is working locally, however you may have a more efficient/effective way to handle this?

File: Casks/q/quickhash.rb [hunk 1]
```diff
@@ -9,9 +9,13 @@
 
   livecheck do
     url "https://www.quickhash-gui.org/downloads/?tax%5Bwpdmcategory%5D=osx/"
-    regex(%r{/quickhash[._-](?:gui[._-])?v?(\d+(?:-\d+)+)[._-]apple[._-]osx/\?wpdmdl=(\d+).*?dt_update_date.+>(\d+(?:/\d+)+)</span}im)
+    regex(%r{
+      dt_update_date[^>]*>\s*(\d+[-/]\d+[-/]\d+)\s*<.+?
```
Comment (bevanjkay) [inline Casks/q/quickhash.rb:13]: @samford Wouldn't using the prior date match check against the previous table row? Potentially matching if the user manual is updated out of sync with the app? Maybe this isn't a likely scenario?

Comment (samford) [reply Casks/q/quickhash.rb:13]: > Wouldn't using the prior date match check against the previous table row?

Unless I'm missing something, the date comes before the URL with the `wpdmdl` query string parameter in a given row:

```html
<tr class="__dt_row">
  <td class="__dt_col_0 __dt_col __dt_col_page_link" style="background-image:url(https://www.quickhash-gui.org/wp-content/plugins/download-manager/assets/file-type-icons/box_download.png);background-size:36px;background-position:5px 8px;background-repeat:no-repeat;padding-left:52px;line-height:normal">
    <a class="package-title" href='https://www.quickhash-gui.org/download/quickhash-gui-v3-3-4-apple-osx/'>Quickhash-GUI v3.3.4 Apple OSX</a>
    <br/>
    <span class='__dt_file_count small-txt'>1</span>
    <span class='__dt_download_count small-txt'>
      <i class="far fa-arrow-alt-circle-down"></i>
       26 downloads
    </span>
    <div class="hidden-md hidden-lg td-mobile"></div>

  </td>
  <td class="__dt_col_1 __dt_col __dt_col_categories">
    <span class='__dt_categories '>
      <a class='sbyc' href='https://www.quickhash-gui.org/downloads/?tax%5Bwpdmcategory%5D=osx'>OSX</a>
    </span>

  </td>
  <td data-order="1698796800" class="__dt_col_2 __dt_col __dt_col_update_date">
    <span class='__dt_update_date '>01/11/2023</span>

  </td>
  <td class="__dt_col_3 __dt_col __dt_col_download_link">
    <a class='wpdm-download-link download-on-click btn btn-primary ' rel='nofollow' href='#' data-downloadurl="https://www.quickhash-gui.org/download/quickhash-gui-v3-3-4-apple-osx/?wpdmdl=2510&refresh=65430c454540f1698892869">Download</a>

  </td>

</tr>
```

Comment (bevanjkay) [reply Casks/q/quickhash.rb:13]: @samford Oh my bad. Sorry about the confusion, I was looking at the link in the first column, which goes to the asset page, not the download URL. Thanks for the fix 🍻️

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**